src/components/appear.js

```diff
@@ -60,18 +60,18 @@ class Appear extends Component {
 
   render() {
     const child = React.Children.only(this.props.children);
-    const endValue = this.state.active ? 1 : 0;
+    const tweenData = this.state.active ? this.props.endValue : this.props.startValue;
```

Comment:
I would be a fan of destructuring `props` to clean this up more. Above line 62, you could destructure `children`, `easing`, `endValue`, and `startValue`. Otherwise, I think this is solid addition, but I'll leave that to the requested reviewers to decide.